### ISSUE TITLE
Parser fails for template with inline SVG containing a viewBox attribute 

### ISSUE THREAD

**Issue by HerringtonDarkholme** [vuejs/eslint-plugin-vue#51]
**Tell us about your environment**

* **ESLint Version:** 4.0.0
* **eslint-plugin-vue Version:** 3.1.3
* **Node Version:** 8.0.0

**Please show your full configuration:**
Vanilla

**What did you do? Please include the actual source code causing the issue.**

```vue
<template>
  <div>
    <svg viewBox="0 0 40 40"></svg>
  </div>
</template>
```

**What did you expect to happen?**
No error reported.

**What actually happened? Please include the actual, raw output from ESLint.**

  0:0  error  Parsing error: Cannot read property 'startOffset' of undefined

https://github.com/octref/vetur/issues/291

**Comment by michalsnik** [comment]
Hey @barraponto. Thanks for pointing this! However I'm going to close this issue as I already answered in #52 and now I'm reading this one and see it's about quite the same thing. I hope you don't mind.

The difference in this case is the case sensitivity instead of prefix existence.

**Comment by michalsnik** [comment]
Fixed in `v3.2.1` 🚀
